Blockzilla/SearchHistoryUtils.swift

```diff
@@ -17,55 +17,37 @@ struct textSearched {
         guard let text = dictionary["text"], let isCurrentSearch = dictionary["isCurrentSearch"] else { return nil }
         self.init(text: text as! String, isCurrentSearch: isCurrentSearch as! Bool)
     }
-
-    var propertyListRepresentation: [String: Any] {
-        return ["text": text, "isCurrentSearch": isCurrentSearch]
-    }
-
 }
 
 class SearchHistoryUtils {
 
     static var isFromURLBar = false
     static var isNavigating = false
     static var isReload = false
+    private static var currentStack = [textSearched]()
```

Comment:
OMG, it pains me that there's a `struct` or `class` with a lowercase name :-(

NOTE: fixing this would be out-of-scope of this patch... was just really confusing for me to read it at first glance